# DP: Use the included sphinx refcounting extension.

Index: b/Doc/conf.py
===================================================================
--- a/Doc/conf.py
+++ b/Doc/conf.py
@@ -13,7 +13,7 @@
 # General configuration
 # ---------------------
 
-extensions = ['sphinx.ext.refcounting', 'sphinx.ext.coverage',
+extensions = ['sphinx.ext.coverage',
               'sphinx.ext.doctest', 'pyspecific']
 templates_path = ['tools/sphinxext']
 
